tests/Shared/RepoTesting/Directory.Packages.Helix.props

```diff
@@ -78,11 +78,6 @@
     <PackageVersion Include="Aspire.StackExchange.Redis" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.StackExchange.Redis.DistributedCaching" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.StackExchange.Redis.OutputCaching" Version="$(PackageVersion)" />
-
-    <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery" Version="$(PackageVersion)" />
```

Comment:
@radical I'm guessing we will need to still have a PackageVersion for these but it will just need to be using the property from Versions.props? It would be good to queue an internal build to make sure this is passing.